src/store/plugins/fungibleTokens.js

```diff
@@ -65,6 +65,9 @@ export default (store) => {
           `${activeNetwork.middlewareUrl}/aex9/by_name`,
         ).catch(handleUnknownError);
 
+        // TODO - remove
+        if (!response?.length || typeof response === 'string') return null;
```

Comment:
I would remove `.catch(handleUnknownError)` from above and use `try/catch` instead

Reply:
- Firstly, i think that was a temprorary fix since the PR is named `WIP...`.
- Secondary, that won't work since 400 and 500 is not an error for the fetch. We need to start throwing and actuall errors on side of fetchJson, but also we need to handle them throug out the whole project.

That's why i don't think this is part of this PR.

Reply:
👌 https://github.com/aeternity/superhero-wallet/issues/1433

I didn't notice `WIP`, used to look at draft/not draft mark